crypto/fips/fips_post.c

```diff
@@ -84,8 +84,6 @@ int FIPS_selftest(void)
         ERR_add_error_data(2, "Type=", "rand_drbg_selftest");
         rv = 0;
     }
-    if (!FIPS_selftest_sha1())
-        rv = 0;
```

Comment:
remove the sha1 selftest from FIPS post. SHA1 is no longer an approved algo for our FIPS offering.